tc/core/polyhedral/cuda/codegen.cc

```diff
@@ -584,12 +592,27 @@ void emitMappedTensorAccess(
 
   // Not promoted, emitting just the mapped subscript.
   if (!promotionInfo.groupId) {
+    // Almost certainly not the proper way but how do we get to a proper
+    // isl::id here?
```

Comment:
Just construct it directly from the name (or use `string`s as keys of the `readOnlySet`)